projects-management/src/main/java/pl/edu/knbit/organizer/events/MemberRejectedEvent.java

```diff
@@ -0,0 +1,25 @@
+package pl.edu.knbit.organizer.events;
+
+import pl.edu.knbit.organizer.MemberId;
+import pl.edu.knbit.organizer.TeamRecruitmentId;
+
+public class MemberRejectedEvent {
+
```

Comment:
I think that it would be better to create abstract event and command associated with TeamRecruitment aggregate root , because every event has teamRecruitmentId and repeated code. You could also introduce hierarchy of events and commands as described here : (http://www.axonframework.org/docs/2.3/single.html#events)